src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
```

Comment:
Hi @AhmedNazirMusah, Based on the requirements of this milestone project, we are being asked to make the App component class-based. Let's kindly comply with the requirements and move a step forward toward the approval of this project.